Antidote/CallsManager.m

```diff
@@ -421,7 +435,7 @@ - (void)switchViewControllerForCall:(OCTCall *)call
 {
     AALogVerbose(@"%@", call);
 
-    if ([self.currentCallViewController isKindOfClass:[ActiveCallViewController class]]) {
+    if ([self isCurrentViewControllerReusableForCurrentCall:call]) {
```

Comment:
This naming is strange. You check if there is reusable controller and then assume that it always is ActiveCallViewController